### PR TITLE
feat: add app_store_choice_screen_engagement_v1 query to firefox_ios_derived

### PR DESCRIPTION
**Pull request by kik-kik**
# feat: add app_store_choice_screen_engagement_v1 query to firefox_ios_derived

## Related Tickets & Documents
* DENG-XXXX
* DSRE-XXXX

**Reviewer, please follow [this checklist](https://github.com/mozilla/bigquery-etl/blob/main/.github/reviewer_checklist.md)**

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-6431)

### CODE REVIEW

**Comment by kik-kik** [comment]
Having some issues running `pip-compile` locally on my machine. This seems to be causing the verify-requirements check to fail. @scholtzan could you try and see if you are able to run the command on your machine?

File: sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_choice_screen_engagement_v1/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+owners:
+- kik@mozilla.com
+labels:
+  schedule: daily
+  incremental: false
+scheduling:
+  dag_name: bqetl_firefox_ios
+  depends_on_past: false
+  arguments:
+  - --date={{ds}}
```
Comment (kik-kik) [inline sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_choice_screen_engagement_v1/metadata.yaml:16]: @scholtzan this should get correctly parametised by Airflow, right?

**Comment by scholtzan** [comment]
bumping the version helped

**Comment by kik-kik** [comment]
> bumping the version helped

Thank you!! 🙇

File: sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_choice_screen_engagement_v1/query.py [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+from google.cloud import bigquery
+
+from bigquery_etl.schema import Schema
+
+#
+# https://developer.apple.com/documentation/AppStoreConnectAPI/downloading-analytics-reports
+#
+# export CONNECT_ISSUER_ID=
+# export CONNECT_KEY_ID=
+# export CONNECT_KEY_FILE_PATH=
```
Comment (scholtzan) [inline sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_choice_screen_engagement_v1/query.py:24]:
```suggestion
```

File: sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_choice_screen_engagement_v1/schema.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+fields:
+
+- mode: NULLABLE
+  name: date
+  type: DATETIME
+  description:
```
Comment (scholtzan) [inline sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_choice_screen_engagement_v1/schema.yaml:6]: might want to add some descriptions here

**Review by scholtzan — APPROVED** [review]

File: sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_choice_screen_engagement_v1/schema.yaml (hunk 3 above)
Comment (kik-kik) [reply sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_choice_screen_engagement_v1/schema.yaml]: done! The plan was to add it at the very end once everything else was in place.

**Review by scholtzan — APPROVED** [review]

**Pull request merged**